SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialInterestConstraints.h

```diff
@@ -0,0 +1,270 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```

Comment:
I believe this shouldn't sit in the EngineClasses folder since none of these classes extend off of an Engine class, like NetDriver or ActorChannel. I think it should sit in the Interop folder.

Reply:
👍